pkg/identity/identity_test.go

```diff
@@ -54,7 +55,29 @@ var _ = Describe("GetCredentials", func() {
 	AfterEach(func() {
 		Expect(k8sclient.Delete(ctx, ns)).To(Succeed())
 	})
+	klog.Infof("with checking if cluster spec identity Ref secret")
```

Comment:
Still unclear around why there are log statements describing the tests that are being run. Like I mentioned before adding log/printf statements for the purpose of reporting is not the best way to move forward.